### DISCUSSION TITLE
slack.linkerd.io returns 520

### DISCUSSION THREAD

**Discussion by noodle-rubrik** [linkerd/linkerd2#8558] [General] [answered]
I'm trying to join the Linkerd Slack to ask a question, and the link isn't working. I'm not sure where else to post this besides here.

https://slack.linkerd.io/ returns a cloudflare 520. External websites seem to show it's not only a problem for me, and that it was working as recently as 2022-05-15.
https://downforeveryoneorjustme.com/slack.linkerd.io?proto=https
https://web.archive.org/web/20220515063036/https://slack.linkerd.io/

**Comment by mateiidavid** [comment, marked as answer]
Hey @noodle-rubrik, sorry for the inconvenience. Not sure what happened, but Linkerd's slack workspace was down for a brief period. It should be back up now, so if you have any questions, please drop by :)

**Comment by ericsuhong** [comment]
@mateiidavid Now it is returning 521 again

**Reply by adleong** [reply]
Thanks for the heads up.  It's back now!
